Apps/WebClient/src/ClientApp/src/views/dependents.vue

```diff
@@ -0,0 +1,70 @@
+<script lang="ts">
+import Vue from "vue";
+import { Component } from "vue-property-decorator";
+import { ILogger } from "@/services/interfaces";
+import { SERVICE_IDENTIFIER } from "@/plugins/inversify";
+import container from "@/plugins/inversify.config";
+import LoadingComponent from "@/components/loading.vue";
+
+@Component({
+    components: {
+        LoadingComponent,
+    },
+})
+export default class DependentsView extends Vue {
+    private logger: ILogger = container.get(SERVICE_IDENTIFIER.Logger);
+    private isLoading = true;
+    private hasErrors = false;
+    private errorMessage = "";
+
+    private handleError(error: string): void {
+        this.hasErrors = true;
+        this.errorMessage = error;
+        this.logger.error(error);
+    }
+
+    private mounted() {
+        this.isLoading = false;
+    }
+}
+</script>
+<template>
+    <div>
+        <LoadingComponent :is-loading="isLoading"></LoadingComponent>
+        <b-row class="my-3 fluid">
+            <b-col class="col-12 col-lg-9 column-wrapper">
+                <b-row>
```

Comment:
has errors is no longer needed since it lives at the App.vue level

Reply:
remvoed